specs/processes.table

```diff
@@ -39,8 +39,8 @@ extended_schema(WINDOWS, [
 extended_schema(DARWIN, [
     Column("upid", BIGINT, "A 64bit pid that is never reused. Returns -1 if we couldn't gather them from the system."),
     Column("uppid", BIGINT, "The 64bit parent pid that is never reused. Returns -1 if we couldn't gather them from the system."),
-    Column("cpu_type", INTEGER, "A 64bit pid that is never reused. Returns -1 if we couldn't gather them from the system."),
-    Column("cpu_subtype", INTEGER, "The 64bit parent pid that is never reused. Returns -1 if we couldn't gather them from the system."),
+    Column("cpu_type", INTEGER, "Indicates the specific processor designed for installation (Type 1 or Type 2)."),
```

Comment:
```suggestion
    Column("cpu_type", INTEGER, "Indicates the specific processor designed for installation."),
```

Reply:
This has a broader range of values

Reply:
Hi @directionless thanks for the review, if I’m not mistaken the changes to be done is to remove the ( Type 1 or Type 2) values and also change INTEGER to STRING?

Reply:
I think we can leave the column type as `INTEGER` and only remove what is suggested here.